### ISSUE TITLE
Issue with content loading in iFrame

### ISSUE THREAD

**Issue by gregatp** [DevExpress/testcafe#5851]
### What is your Test Scenario?

We are testing H5P integration with our service.
Tests scripts works fine in TestCafe v1.9.4
but when we upgrade to the latest version TestCafe v1.10.1
adding H5P content using CKEditor starts failing.
 
### What is the Current behavior?

switchToIframe() function call displays the following error:
```sh
Content of the iframe to which you are switching did not load.
```
 
### What is the Expected behavior?

Context is switched to the selected CKEditor iframe and text is typed in it.
 
### What is your web application and your TestCafe test code?

 
<details>
<summary>Your website URL (or attach your complete example):</summary>

https://tciframe1.h5p.com/content/create
</details>

<details>
<summary>Your complete test code (or attach your test files):</summary>

 
```js
import {Selector} from 'testcafe';

fixture`My fixture`
    .page`https://tciframe1.h5p.com/content/create`;

const user = '<your email>@gmail.com';
const passwd = '<your password>';

const loginInput = Selector('#email');
const passwdInput = Selector('#password');
const loginSubmit = Selector('button[type="submit"]');

const iFrame = Selector('iframe.h5p-editor-iframe');
const contentTypeQuiz = Selector('#h5p-questionset button[type="button"]');
const btnUse = Selector('button.button-use');
const questionType = Selector('#field-question-37');
const questionInput = Selector('.field-name-question .ckeditor');
const questionInputIframe = Selector('.field-name-question iframe');
const questionInputCkeditor = Selector('body p');

test('Check iFrame without "src"', async t => {
    await t
        .typeText(loginInput, user)
        .click(loginSubmit)
        .typeText(passwdInput, passwd)
        .click(loginSubmit);
    await t
        .switchToIframe(iFrame);
    await t
        .click(contentTypeQuiz)
        .click(btnUse)
        .click(questionType)
        .click(questionType.find('option').withAttribute('value', /H5P.MultiChoice/))
        .click(questionInput)
        .click(questionInputIframe);
    await t
        .switchToIframe(questionInputIframe);
    await t
        .typeText(questionInputCkeditor, 'Sample question');

    await t.debug();
}); 
```
</details>

<details>
<summary>Your complete configuration file (if any):</summary>

```

```
</details>

<details>
<summary>Your complete test report:</summary>

```
 ✖ Check iFrame without "src"

   1) Content of the iframe to which you are switching did not load.

      Browser: Chrome 87.0.4280.88 / macOS 10.15.7

         32 |        .click(questionType)
         33 |        .click(questionType.find('option').withAttribute('value', /H5P.MultiChoice/))
         34 |        .click(questionInput)
         35 |        .click(questionInputIframe);
         36 |    await t
       > 37 |        .switchToIframe(questionInputIframe);
         38 |    await t
         39 |        .typeText(questionInputCkeditor, 'Sample question');
         40 |
         41 |    await t.debug();
         42 |});
```
</details>

<details>
<summary>Screenshots:</summary>

![iframe](https://user-images.githubusercontent.com/56645176/103896821-d8878580-50f2-11eb-9234-318f986b7ca0.png)
</details>

### Steps to Reproduce:

run the test script attached above (you can create your trial app in https://h5p.com/ service, it is free for 30 days)

### Your Environment details:
 
* testcafe version:                    1.10.1
* node.js version:                     v13.14.0
* command-line arguments:   yarn testcafe chrome tests/iframe.js
* browser name and version:  Chrome 87.0.4280.88
* platform and version:           macOS 10.15.7
* other:

**Comment by AlexSkorkin** [comment]
We received your access information. However, we also require written access confirmation. I've sent additional details to your email.

**Comment by gregatp** [comment]
@AlexSkorkin I've send my credentials just in case, but you can setup your own instance of H5P app, free trial is available.

**Comment by Farfurix** [comment]
@gregatp 

Hello,

I created my own account and reproduced the issue on Windows 10 / Chrome 87. Our team will examine this issue and check for a suitable solution.

**Comment by gregatp** [comment]
This works fine in version v1.14.0. 
It seems that it has been fixed as side effect of release v1.11.0
